task7/index.js

```diff
@@ -0,0 +1,39 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const div = document.getElementById("js-loading");
+const gif = document.createElement('img');
+div.appendChild(gif);
+const loading = () => {
+    gif.src = "loading-circle.gif";
+};
+
+const getData = new Promise((resolve) => {
+    loading();
+
+    setTimeout(() => {
+        resolve(obj);
+        gif.remove();
```

Comment:
It is not appropriate to delete the image here. It's better to delete them in .then.

Reply:
So that's it.
This doesn't mean that when you receive the passed value, you delete it.